Dragon Valley rework after play testing (#234)
qwrtln/Homm3BG-mission-book: draft-scenarios/clash/dragon_valley.tex

> 0% !TeX spellcheck = en_US

Reviewer: ```suggestion
% !TeX spellcheck = en_US
```

\textbf{Starting Units:}

\begin{itemize}
  \item A Pack of cheapest \bronze\ Units

>   \item A Few  of chosen \bronze\ Units

Reviewer: ```suggestion
  \item A Few chosen \bronze\ Units
```
"Pack of", but "few" does not need a preposition.

\end{itemize}

\textbf{Town Buildings:}
\begin{itemize}
  \item \bronze\ Dwelling
  \item Each player may build the \svg{building_special_tent} building at a discount of 4 \svg{gold} and 2 \svg{building_materials}.
\end{itemize}

\subsection*{\MakeUppercase{Map Setup}}
Take the following Map Tiles ($P$ stands for the number of players) and arrange them as shown in the Scenario map layout:

\begin{itemize}
  \item P × Starting (I) Map Tile
  \item 3P × Far (II–III) Map Tiles
  \item 2P × Near (IV–V) Map Tiles
  \item P × Center (VI–VII) Map Tile
  \item 1 × Center (VI–VII) Map Tile with a Dragon Utopia
\end{itemize}

\textbf{Additionally, for 4 players:}
\begin{itemize}
  \item 1 × Center (VI–VII) Map Tile with a Dragon Utopia
\end{itemize}

\subsection*{\MakeUppercase{Victory Conditions}}
The game ends at the end of the Round when any of the following conditions are met:

\begin{itemize}

>  \item One player has defeated each other player's Main Heroes once. -- \textit{That player wins the game immediately.}

Reviewer: ```suggestion
 \item One player has defeated each other player's Main Heroes once. \textit{That player wins the game immediately.}
```
Shouldn't those dashes be mid-sentence?

>  \item One player has conquered both Dragon Utopias. -- \textit{That player wins the game immediately.}

Reviewer: ```suggestion
 \item One player has conquered both Dragon Utopias. \textit{That player wins the game immediately.}
```

 \item At the end of Round 13.
\end{itemize}

> If no player achieves immediate victory, the player controlling a Dragon Utopia wins. If different players control the two Utopias, they each draw Cards up to their Hand Limit and engage in a final Combat - the winner is the victor. If no one controls a Dragon Utopia, all players lose.

Reviewer: ```suggestion
If no player achieves immediate victory, the player controlling a Dragon Utopia wins.
If different players control the two Utopias, they each draw Cards up to their Hand Limit and engage in a final Combat -- the winner is the victor.
If no one controls a Dragon Utopia, all players lose.
```

\subsection*{\MakeUppercase{Timed Events}}

\textbf{\nth{2} Round:}
\begin{itemize}

>   \item Recruit one Unit that has died for free.

Reviewer: Faction or neutral? Or any?

Author: oh yeah, that needs "one of your units that died.."

\end{itemize}
\textbf{\nth{4} Round:}
\begin{itemize}
  \item Remove all Black Cubes.
\end{itemize}
\textbf{\nth{8} Round:}
\begin{itemize}
  \item Remove all Black Cubes.
\end{itemize}
\textbf{End of \nth{10} Round:}
\begin{itemize}

>   \item The Player(s) whose Main Hero has the least experience rolls 2 \svg{resource}.

Reviewer: ```suggestion
  \item The Player(s) whose Main Hero has the least experience roll(s) 2 \svg{resource}.
```
Nitpick ;-)

Author: oh btw. should it be lower case "player"? looks like I forgot The capitol P after adding "The" infront

\end{itemize}

\subsection*{\MakeUppercase{Additional Rules}}
\begin{itemize}

>   \item \textbf{Grail Token}: When a player gains or starts their turn with it, gain +1 \svg{morale_positive} and Search (2) \svg{spellpower} or \svg{artifact}. Both Main and Secondary Heroes may gain the Grail Token.

Reviewer: ```suggestion
  \item \textbf{Grail Token}: When a player gains or starts their turn with it, they gain +1 \svg{morale_positive} and Search (2) \svg{spellpower} or \svg{artifact}. Both Main and Secondary Heroes may gain the Grail Token.
```

Reviewer: I'm not sure if the differentiation here is important, but maybe the player gains the grail, but the hero picks it up?

Author: yea that looks good 👍

  \item Level VII Neutral Combats cannot be skipped.
  \item A defeated Main Hero may Empower one Defense Statistic Card from his M\&M Deck and gain +1 \svgeven{movement} for the next Round.
  \item After winning a Combat against another player, draw up to your Hand Limit and discard 1 Card.
  \item Level VII \textbf{Settlements/Random Town}: Increase all incomes by 1 space and Draw 2 Cards.
  \item \textbf{Dragon Utopias} are guarded only by dragons.

>   \item Upon flagging a \textbf{Dragon Utopia}, choose one and then put a Black Cube:

Reviewer: ```suggestion
  \item Upon flagging a \textbf{Dragon Utopia}, choose one option and then put a Black Cube:
```

  \begin{itemize}

>     \item Sacrifice any number of Units, then recruit one of the defeated dragons at a discount equal to the total recruitment cost of the sacrificed Units.

Reviewer: ```suggestion
    \item Sacrifice any number of Units, then recruit one of the defeated dragons at a discount equal to the total recruitment cost of the sacrificed Units (to a minimum of 0).
```

    \item Gain 15 \svg{gold} and Search (3) \svg{spellpower} or \svg{artifact}.
  \end{itemize}
\end{itemize}

\begin{center}
  \vspace*{\fill}
  \includegraphics[width=1.0\linewidth]{\maps/dragon-valley-2p.png}
  \captionof{figure}{\textbf{2-PLAYER SCENARIO}}
\end{center}

\begin{itemize}

>   \item When a \textbf{Secondary Hero} is recruited, you may gain a Crown Token and one of each: Attack, Defense, Power and Magic Arrow. These may only be used by your Secondary Hero.

Reviewer: ```suggestion
  \item When a \textbf{Secondary Hero} is recruited, you gain an additional Empowerment and one of each: Attack, Defense and Power Statistic Cards, and a Magic Arrow. These may only be used by your Secondary Hero.
```
There are no crown tokens in the game, at least not yet. Where do those cards go? Do you form a separate deck?

Author: yeah it would be a separate deck, but you would always draw back all four of those and wouldnt add anything to it. I have to keep thinking about tho rule

>   \item When a Secondary Hero starts their turn on a I or II–III Map Tile, that Hero gains +1 \svgeven{movement}.

Reviewer: ```suggestion
  \item When a Secondary Hero starts their turn on a I or II--III Map Tile, they gain +1~\svgeven{movement}.
```
